QuantConnect.CoinbaseBrokerage/CoinbaseBrokerage.Messaging.cs

```diff
@@ -44,7 +46,27 @@ public partial class CoinbaseBrokerage
         /// Represents a rate limiter for controlling the frequency of WebSocket operations.
         /// </summary>
         /// <see cref="https://docs.cloud.coinbase.com/advanced-trade-api/docs/ws-rate-limits"/>
-        private RateGate _webSocketRateLimit = new(750, TimeSpan.FromSeconds(1));
```

Comment:
Is it not 750 requests/second?

Reply:
Used previous value from old api.
I don't know properly, cuz the documentation is messy.